.travis.yml

```diff
@@ -21,7 +21,7 @@ before_script:
 
 script: 
   - mvn -B clean install -DskipITs=false
-  - mvn -B clean -DskipITs=false cobertura:cobertura cobertura:cobertura-integration-test coveralls:report
+  - mvn -B clean -DskipITs=false -DCONSONANCE_LOG_FILE=./arch3_test.log cobertura:cobertura cobertura:cobertura-integration-test coveralls:report
```

Comment:
Is it necessary to change the default build command here? 
Does the default of `test.log` work for those surefire plugins?